src/reference/asciidoc/aggregator.adoc

```diff
@@ -89,66 +81,54 @@ public AbstractCorrelatingMessageHandler(MessageGroupProcessor processor, Messag
+Consequently, the `Collection<Message>` variable in the POJO is cleared too, if it is passed out of the aggregator.
+If you wish to simply release that collection as-is for further processing, you must build a new `Collection` (for example, `new ArrayList<Message>(messages)`).
+Starting with version 4.3, the framework no longer copies the messages to a new collection, to avoid undesired extra object creation.
 
 
-If the `MessageGroupProcessor` 's `processMessageGroup` method returns a collection, it must be a collection of
-`Message<?>` s.
-In this case, the messages are released individually.
-Prior to _version 4.2_, it was not possible to provide a `MessageGroupProcessor` using XML configuration, only POJO
-methods could be used for aggregation.
-Now, if the framework detects that the referenced (or inner) bean implements `MessageProcessor`, it is used as the
-aggregator's output processor.
+If the `processMessageGroup` method of the `MessageGroupProcessor` returns a collection, it must be a collection of `Message<?>` objects.
+In this case, the messages are individually released.
+Prior to version 4.2, it was not possible to provide a `MessageGroupProcessor` by using XML configuration.
+Only POJO methods could be used for aggregation.
+Now, if the framework detects that the referenced (or inner) bean implements `MessageProcessor`, it is used as the aggregator's output processor.
 
-If you wish to release a collection of objects from a custom `MessageGroupProcessor` as the payload of a message, your
-class should extend `AbstractAggregatingMessageGroupProcessor` and implement `aggregatePayloads()`.
+If you wish to release a collection of objects from a custom `MessageGroupProcessor` as the payload of a message, your class should extend `AbstractAggregatingMessageGroupProcessor` and implement `aggregatePayloads()`.
 
-Also, since _version 4.2_, a `SimpleMessageGroupProcessor` is provided; which simply returns the collection of
-messages from the group, which, as indicated above, causes the released messages to be sent individually.
+Also, since version 4.2, a `SimpleMessageGroupProcessor` is provided.
+It returns the collection of messages from the group, which, as indicated earlier, causes the released messages to be sent individually.
 
-This allows the aggregator to work as a message barrier where arriving messages are held until the release strategy
-fires, and the group is released, as a sequence of individual messages.
+This lets the aggregator work as a message barrier, where arriving messages are held until the release strategy fires and the group is released as a sequence of individual messages.
 
-===== ReleaseStrategy
+===== `ReleaseStrategy`
 
 The `ReleaseStrategy` interface is defined as follows:
 
+====
```

Comment:
Looks like you wrapped almost all the code snippets to these symbols.
What is the point, please ?

Reply:
For code listings with callouts, it groups the callouts in a box with the code listing. Because that applies a certain formatting, we also need to do it all the other code listings. So, yes, I try to do it to all of them. Missing one would be the error.